server/drivers/CFontzPacket.c

```diff
@@ -149,38 +148,35 @@ CFontzPacket_init (Driver *drvthis)
 	EmptyKeyRing(&keyring);
 	EmptyReceiveBuffer(&receivebuffer);
 
-	/* Read config file */
+	/* Read config */
+	CFontzPacketDriverConfig config;
+	elektraGet2V(elektra, &config, ELEKTRA_TAG_CFONTZPACKET, drvthis->index);
```

Comment:
2V?

Reply:
`V` is for the vararg version.

`2` is just because I didn't know what to call the version of `elektraGet` that uses a result pointer instead of returning a value. This is version of `elektraGet` only exists for structs and doesn't malloc.

Reply:
> 2 is just because I didn't know what to call the version of elektraGet that uses a result pointer instead of returning a value

Please always create issues for name discussions in such situations. If such names creep into the API you never get rid of them again.

Please do so now for API names where you have doubts that they are perfect.